### PR TITLE
Refactor: List, search, logo

### PR DESCRIPTION
**Pull request by birloiflorian**
C4R-1255, C4R-1040, C4r-1256, C4R-1277, C4r-1346, C4r-1348, C4r-1280

### CODE REVIEW

**Comment by radulescuandrew** [comment]
🤮

File: src/common/components/breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
 import { NavLink, useLocation } from 'react-router-dom';
-import useBreadcrumbs from "use-react-router-breadcrumbs";
-import { useSelectedOrganization, useSelectedPracticeProgram } from '../../../store/Selectors';
+import useBreadcrumbs from 'use-react-router-breadcrumbs';
 import i18n from '../../configs/i18n';
 import { MENU_ROUTES_HREF } from '../../constants/Menu.constants';
 
 const DynamicPracticeProgram = ({ match }: any) => {
-  const { selectedProgram } = useSelectedPracticeProgram();
+  // const { selectedProgram } = useSelectedPracticeProgram();
```
Comment (radulescuandrew) [inline src/common/components/breadcrumbs/Breadcrumbs.tsx:9]: Lot of comments

File: src/common/components/breadcrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
 
   return (
-    <span>{selectedProgram?.title || match.params.id}</span>
-  )
+    // <span>{selectedProgram?.title || match.params.id}</span>
+    <span>{match.params.id}</span>
```
Comment (radulescuandrew) [inline src/common/components/breadcrumbs/Breadcrumbs.tsx:13]: Why not displaying title?

File: src/common/components/breadcrumbs/Breadcrumbs.tsx [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
 
 const DynamicOrganization = ({ match }: any) => {
-  const { selectedOrganization } = useSelectedOrganization();
+  // const { selectedOrganization } = useSelectedOrganization();
 
-  return (
-    <span>{selectedOrganization?.name || match.params.id}</span>
-  )
+  // return <span>{selectedOrganization?.name || match.params.id}</span>;
+  return <span>{match.params.id}</span>;
```
Comment (radulescuandrew) [inline src/common/components/breadcrumbs/Breadcrumbs.tsx:21]: Why not displaying title?

File: src/common/containers/Layout.tsx [hunk 4]
```diff
@@ -10,11 +10,11 @@ const Layout = () => {
...
 
   return (
-    <div className="w-screen h-screen max-w-full">
+    <div className="w-screen max-w-full">
       <Header openSlidingMenu={setSlidingMenuOpen} />
       <SlidingMenu isOpen={isSlidingMenuOpen} setSlidingMenuOpen={setSlidingMenuOpen} />
-      <div className="content flex flex-col w-full">
-        <Breadcrumbs />
+      <div className="flex flex-col w-full">
+        {/* <Breadcrumbs /> */}
```
Comment (radulescuandrew) [inline src/common/containers/Layout.tsx:17]: Comment

File: src/pages/organizations/Organizations.tsx [hunk 5]
```diff
@@ -1,76 +1,69 @@
...
+            listClassName="virtuso-grid-list"
+            components={{
+              Footer: () => (
+                <InfiniteScrollFooter
+                  hasNoData={data?.pages?.length === 0}
+                  isLoading={isFetching}
+                />
+              ),
+              Header: () => {
+                return data?.pages.length !== 0 && !isFetching ? (
```
Comment (radulescuandrew) [inline src/pages/organizations/Organizations.tsx:51]: Merge scos intr-o componenta

File: src/pages/programs/Programs.tsx [hunk 6]
```diff
@@ -1,78 +1,71 @@
...
-    query.page as number,
-    query.search,
-    query.locationId,
-    query.faculties,
-    query.workingHours as WorkingHoursEnum,
-    query.domains,
-    query.start,
-    query.end,
-  );
+  const [query] = useQueryParams(POGRAMS_QUERY_PARAMS);
```
Comment (radulescuandrew) [inline src/pages/programs/Programs.tsx:18]: Pograms <3

File: src/store/Store.ts [hunk 7]
```diff
@@ -2,29 +2,7 @@ import create from 'zustand';
...
 import { Faculty } from '../common/interfaces/Faculty.interface';
-import { Organization } from '../common/interfaces/Organization.interface';
-import { OrganizationFlat } from '../common/interfaces/OrganizationFlat.interface';
-import { PaginatedEntity } from '../common/interfaces/PaginatedEntity.interface';
-import { IPracticeProgram } from '../common/interfaces/PracticeProgram.interface';
 import { nomenclatureSlice } from './nomenclatures/Nomenclatures.slice';
-import { organizationsSlice } from './organizations/Organizations.slice';
-import { practiceProgramsSlice } from './practice-programs/PracticePrograms.slice';
-
-interface PracticeProgramsState {
```
Comment (radulescuandrew) [inline src/store/Store.ts:13]: Wow, de cat state scapam

**Review by radulescuandrew — CHANGES_REQUESTED** [review]

**Comment by MarG66** [comment]
PR Test Results:
Not Run
[1040](https://wearetribus.atlassian.net/browse/C4R-1040) - Unable to reproduce

Passed
[1255](https://wearetribus.atlassian.net/browse/C4R-1255)
[1256](https://wearetribus.atlassian.net/browse/C4R-1256) - new visual bug discovered 
[1277](https://wearetribus.atlassian.net/browse/C4R-1277)
[1346](https://wearetribus.atlassian.net/browse/C4R-1346)
[1348](https://wearetribus.atlassian.net/browse/C4R-1348)
[1280](https://wearetribus.atlassian.net/browse/C4R-1280)

Failed
[1292](https://wearetribus.atlassian.net/browse/C4R-1292) - NGO logo still not displayed, it got changed to what is shown in Figma
[1330](https://wearetribus.atlassian.net/browse/C4R-1330)
[1260](https://wearetribus.atlassian.net/browse/C4R-1260)

File: src/common/components/breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (birloiflorian) [reply src/common/components/breadcrumbs/Breadcrumbs.tsx:9]: https://wearetribus.atlassian.net/browse/C4R-1514

File: src/common/components/breadcrumbs/Breadcrumbs.tsx (hunk 2 above)
Comment (birloiflorian) [reply src/common/components/breadcrumbs/Breadcrumbs.tsx:13]: https://wearetribus.atlassian.net/browse/C4R-1514

File: src/common/components/breadcrumbs/Breadcrumbs.tsx (hunk 3 above)
Comment (birloiflorian) [reply src/common/components/breadcrumbs/Breadcrumbs.tsx:21]: https://wearetribus.atlassian.net/browse/C4R-1514

File: src/common/containers/Layout.tsx (hunk 4 above)
Comment (birloiflorian) [reply src/common/containers/Layout.tsx:17]: https://wearetribus.atlassian.net/browse/C4R-1514

**Pull request merged**